### ISSUE TITLE
[Feature]: A key for extra-keys which opens a menu to select a less commonly used `supported keys`

### ISSUE THREAD

**Issue by sudomain** [termux/termux-app#2649]
### Feature description

There's supported keys that I (and possibly other users) need occasionally, but don't want to commit them to my `termux.properties`. For me these are the `F1-F12` and `FN` keys. I could technically add them to my termux.properties, run `termux-reload-settings`, press them, and go back to my old config when I'm done. I think it would be nicer to have a key/popup/gesture which opens a menu displaying the list of the supported keys. When one of the menu items is tapped, that key gets sent. It would be useful for quickly accessing the supported keys that are not used very frequently (at least compared to the reload-settings process previously described).

### Additional information

N/A

**Comment by agnostic-apollo** [comment]
Your request is different but have you tried `popup` keys which trigger on swipe up gesture and don't show otherwise.

https://wiki.termux.com/wiki/Touch_Keyboard#Extra_Keys_Row

**Comment by sudomain** [comment]
Yup, but it's my understanding that only 1 popup is allowed per key. This menu might be easier to implement than #2624 ?
